supervisord/server.conf

```
@@ -1,25 +1,5 @@
-[unix_http_server]
-file = /tmp/supervisord/supervisor.sock
-
-[supervisorctl]
-serverurl = unix:///tmp/supervisord/supervisor.sock
-
-
-[rpcinterface:supervisor]
-supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
-
-[supervisord]
-nodaemon=true
-logfile=%(ENV_HOME)s/logs/supervisord.log ; supervisord log file
-logfile_maxbytes=50MB       ; maximum size of logfile before rotation
-logfile_backups=10          ; number of backed up logfiles
-loglevel=debug              ; info, debug, warn, trace
-pidfile=/tmp/supervisord/supervisord.pid ; pidfile location
-
-[program:clamav_update]
-startsecs=0
-command=bash -c "if [ \"${CLAM_AV}\" = 'yes' ]; then /usr/bin/freshclam -d \
-    -l %(ENV_HOME)s/logs/freshclam.log --foreground=true; fi"
+[include]
+files = reusable/supervisord.conf reusable/clamav.conf reusable/smokescreen.conf
```

```suggestion
files = reusable/supervisord.conf %(ENV_CVAT_SUPERVISORD_INCLUDES)s
```

The includes have to be supplied by the entrypoint script, because clamav may not need to be included.